docgen/src/guide/Virtual_widgets.md

```diff
@@ -28,6 +29,7 @@ const App = () =>
     <Menu attributeName="fruits" defaultRefinement="Orange" />
   </InstantSearch>;
 ```
+In this case, we are using the `VirtualMenu` to pre refine our results (within the `clothes` search, only display `hoodies`). Think of the `VirtualMenu` as a hidden filter where we define attributes and values that will always apply the our search results.
```

Comment:
Suggestion: As a mean to impose a parameter chosen beforehand?

Reply:
@marielaures I am sorry, but I am not sure I understand what you are trying to say. Can you elaborate?

Reply:
@dericcain I was trying to rephrase the second sentence ("hidden filter where we define attributes and values..." because  "pre refine" might be a bit confusing! But I agree there it's not easy :)